snuba/web/db_query.py

```diff
@@ -251,7 +254,10 @@ def raw_query(
         if isinstance(cause, RateLimitExceeded):
             stats = update_with_status("rate-limited")
         else:
-            logger.exception("Error running query: %s\n%s", sql, cause)
+            with configure_scope() as scope:
+                if isinstance(cause, ClickhouseError):
+                    scope.fingerprint = ["{{default}}", str(cause.code)]
```

Comment:
Are you sure. scope cannot be None (especially if you do not have the sdk set up)?

Other than this, I don't think it can. break anything:
```
@_attr_setter
    def fingerprint(self, value):
        # type: (Optional[List[str]]) -> None
        """When set this overrides the default fingerprint."""
        self._fingerprint = value
```

Reply:
Yeah It cannot be none

Reply:
This seems to indicate that if a context manager is returned, `Scope` is non-Optional:

https://github.com/getsentry/sentry-python/blob/7cdd713791a516a39e317c9eaddf0cc7cf0c8fc7/sentry_sdk/api.py#L113-L116

https://github.com/getsentry/sentry-python/blob/7cdd713791a516a39e317c9eaddf0cc7cf0c8fc7/sentry_sdk/api.py#L127-L144